src/containers/MyNdla/Folders/FoldersPage.tsx

```diff
@@ -245,18 +245,16 @@ const FoldersPage = () => {
         )}
         <ListViewOptions type={viewType} onTypeChange={setViewType} />
       </StyledRow>
-      {!!folders.length && (
-        <FolderList
-          onFolderAdd={onFolderAdd}
-          isAdding={isAdding}
-          setIsAdding={setIsAdding}
-          type={viewType}
-          folders={folders}
-          loading={loading}
-          folderId={folderId}
-          setFolderAction={setFolderAction}
-        />
-      )}
+      <FolderList
+        onFolderAdd={onFolderAdd}
+        isAdding={isAdding}
+        setIsAdding={setIsAdding}
+        type={viewType}
+        folders={folders}
+        loading={loading}
+        folderId={folderId}
+        setFolderAction={setFolderAction}
+      />
```

Comment:
Jeg kan ikke forklare hvorfor, men denne endringen gjør at sortering ikke fungerer for mapper.

Testa med `/minndla/folders/e80a1a15-4d15-4190-b77d-f785dc9289bb` hos daniel laerervgs